### ISSUE TITLE
Using from the Android settings dialog

### ISSUE THREAD

**Issue by sthewissen** [SirJohnK/LocalizationResourceManager.Maui#32]
I wanted to use the NuGet to implement translations, so far so good. However, I didn't want to roll my own select language dialog so for Android I added a `xml/locales_config.xml` file:

```xml
<?xml version="1.0" encoding="utf-8" ?>
<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
	<locale android:name="en-US"/>
	<locale android:name="nl"/>
</locale-config>
```

Then I linked that in my manifest:

```xml
<application android:allowBackup="true" 
	 android:icon="@mipmap/appicon" 
	 android:roundIcon="@mipmap/appicon_round" 
	 android:supportsRtl="true" 
	 android:localeConfig="@xml/locales_config">	
```

I have a button that pops open the settings dialog with `AppInfo.ShowSettingsUI()`. When selecting a different language there and going back into the app it stays on the old language. A full app reboot DOES change the language. Is there something I'm missing or something built into the NuGet that would be able to handle this scenario?

**Comment by SirJohnK** [comment]
Yes, in all scenarios the ```ILocalizationResourceManager.CurrentCulture``` need to be set to trigger application language refresh. In this case, it needs to be set after the ```Locale``` has been set in ```Settings```. 

After some quick testing, I came up with this solution that works, regardless where in the app this is triggered:
```csharp
using System.Globalization;

namespace LocalizationResourceManager.Maui.Sample
{
    public partial class App : Application
    {
        private readonly ILocalizationResourceManager localizationResourceManager;

        public App(IServiceProvider serviceProvider)
        {
            InitializeComponent();

            localizationResourceManager = serviceProvider.GetRequiredService<ILocalizationResourceManager>();

            MainPage = new AppShell();
        }

        protected override Window CreateWindow(IActivationState? activationState)
        {
#if ANDROID
            localizationResourceManager.CurrentCulture = CultureInfo.GetCultureInfo(Java.Util.Locale.Default.Language);
#endif
            var window = base.CreateWindow(activationState);
            return window;
        }
    }
}
```

Maybe a more elegant solution would be to have a ```Locale``` change listener that trigger the ```CurrentCulture``` change and something similar for other platforms. 😄 

Note: That the ```RestoreLatestCulture``` should not be activate in this case, since that will conflict with setting the language outside the application.

If you find any better solution for this or anything that could be part of the library, I would love your feedback!

Finally I wanted to thank you for trying out my library and it was nice seeing you at the ```.NET MAUI Day``` in Cologne in February! 👍

**Comment by sthewissen** [comment]
I'll play around with this some more, thanks for that! I'm happy to try it, it's a great library 👍 Nice seeing you indeed! 👌
